Dockerfile-csi-controller.test

```diff
@@ -30,11 +30,11 @@ RUN pip3 install -r /driver/controller/requirements-tests.txt
 
 COPY ./common /driver/common
 COPY ./controller /driver/controller
+USER root
```

Comment:
This is build stage right? So it is root on  our system?

Reply:
yes, ubi default user is non-root, and we need root to chown the `/driver` dir to `appuser`